### DISCUSSION TITLE
How can I help you for the French translation?

### DISCUSSION THREAD

**Discussion by Yellowbery01** [usememos/memos#2911] [Ideas]
I am not aware of coding
How can I help you for the French translation??

**Comment by boojack** [comment]
Thanks for the help. I think the easiest way is to refer to [en.json](https://github.com/usememos/memos/blob/main/web/src/locales/en.json) and then modify [fr.json](https://github.com/usememos/memos/blob/main/web/src/locales/fr.json) accordingly.
